### PR TITLE
Update Loader

### PR DESCRIPTION
**Pull request by mriverre8**

### CODE REVIEW

**Comment by jorbush** [comment]
Should accept the same parameters, the only thing is that them should be initialized by default on the `Loader` component. Not only hardcode them directly.

File: src/components/loader/Loader.jsx [hunk 1]
```diff
@@ -1,22 +1,14 @@
...
-
-const Loader = ({ height, width, barColor, borderColor }) => {
-    Loader.propTypes = {
-        height: PropTypes.number.isRequired,
-        width: PropTypes.number.isRequired,
-        barColor: PropTypes.string.isRequired,
-        borderColor: PropTypes.string.isRequired,
-    };
 
+const Loader = () => {
```
Comment (jorbush) [inline src/components/loader/Loader.jsx:3]: Should accept the same parameters, the only thing is that them should be initialized by default on the `Loader` component. Not only hardcode them directly.

**Review by jorbush — CHANGES_REQUESTED** [review]

File: src/components/loader/Loader.jsx (hunk 1 above)
Comment (mriverre8) [reply src/components/loader/Loader.jsx]: Done

File: src/components/loader/Loader.jsx [hunk 2]
```diff
@@ -1,14 +1,6 @@
...
-
-const Loader = ({ height, width, barColor, borderColor }) => {
-    Loader.propTypes = {
-        height: PropTypes.number.isRequired,
-        width: PropTypes.number.isRequired,
-        barColor: PropTypes.string.isRequired,
-        borderColor: PropTypes.string.isRequired,
-    };
 
+const Loader = ({ height = "47", width = "47", barColor = "#0789c2", borderColor = "#3dc2f3" }) => {
```
Comment (jorbush) [inline src/components/loader/Loader.jsx:3]: Fixed!

**Review by jorbush — APPROVED** [review]
LGTM 😎

**Pull request merged**